pkg/services/accesscontrol/evaluator/evaluator.go

```diff
@@ -9,18 +9,24 @@ import (
 	"github.com/grafana/grafana/pkg/services/accesscontrol"
 )
 
-// Evaluate evaluates access to the given resource, using provided AccessControl instance
+// Evaluate evaluates access to the given resource, using provided AccessControl instance.
+// Scopes are evaluated with an `OR` relationship.
 func Evaluate(ctx context.Context, ac accesscontrol.AccessControl, user *models.SignedInUser, permission string, scope ...string) (bool, error) {
-	res, err := ac.GetUserPermissions(ctx, user)
+	userPermissions, err := ac.GetUserPermissions(ctx, user)
 	if err != nil {
 		return false, err
 	}
 
-	ok, dbScopes := extractPermission(res, permission)
+	ok, dbScopes := extractPermission(userPermissions, permission)
```

Comment:
My two cents on this again extractScopes
```suggestion
	ok, dbScopes := extractScopes(userPermissions, action)
```